controllers/articles.controller.js

```diff
@@ -37,4 +39,33 @@ const getArticleComments = (req, res, next) => {
     });
 };
 
-module.exports = { getArticles, getArticlesById, getArticleComments };
+const postArticleComment = (req, res, next) => {
+  const articleId = req.params.article_id;
+  const username = req.body.username;
+  const commentObj = req.body;
+  const commentBody = req.body.body;
+  if (username && commentBody) {
+    checkUserExists(username)
+      .then(() => {
+        return selectArticlesById(articleId);
```

Comment:
same goes for articleID in that you input into comments table, it has to exist in the articles table as it is a foreign key.